yasnippet.el

```diff
@@ -4487,11 +4493,13 @@ When multiple expressions are found, only the last one counts."
       (when brand-new-mirror
         (push brand-new-mirror
               (yas--field-mirrors field))
-        (yas--calculate-mirrors-in-fields snippet brand-new-mirror)
+        (yas--calculate-simple-fom-parentage snippet brand-new-mirror)
         (push (cons (match-beginning 0) real-match-end-0) yas--dollar-regions)))))
 
-(defun yas--simple-mirror-parse-create (snippet)
-  "Parse the simple \"$n\" fields/mirrors/exitmarkers in SNIPPET."
+(defun yas--simple-fom-create (snippet &optional parent-field)
+  "Parse the simple \"$n\" fields/mirrors/exitmarkers in SNIPPET.
+When PARENT-FIELD is non-nil search 
```

Comment:
It looks like you started adding `parent-field` but ended up not using it.

Reply:
Yep. I squashed a fix.